client/scripts/controllers/chain/cosmos/governance.ts

```diff
@@ -76,11 +76,12 @@ class CosmosGovernance extends ProposalModule<
             if (props.length === 0) {
               return of(props);
             } else {
-              return forkJoin(props.map((p) => p.initialized$)).pipe(map(() => props));
+              return of(props);
+              // return forkJoin(props.map((p) => p.initialized$)).pipe(map(() => props));
```

Comment:
This was needed to get the API to initialize. We were filtering for `initialized` before? I don't fully understand how that worked...

Reply:
the `forkJoin` here forces a wait for all the proposals to initialize -- the `initialized$` is an observable that completes once initialized.

Reply:
Aha, okay, this could be something to test out reverting during your QA pass. I don't recall why it was breaking